config/config.template.ini

```diff
@@ -1,11 +1,9 @@
 [ftp]
 #serverHost = @SERVER_HOST@

```

Comment:
Please remove the comments, the template should be valid after replacing the tokens without the need to uncomment it.